modules/nw-creating-project-and-service.adoc

```diff
@@ -6,44 +6,59 @@
 [id="nw-creating-project-and-service_{context}"]
 = Creating a project and service
 
-If the project and service that you want to expose do not exist, first create
-the project, then the service.
+If the project and service that you want to expose do not exist, you must create the project and then the service.
 
-If the project and service already exist, skip to the procedure on exposing the
-service to create a route.
+If the project and service already exist, go the "Exposing the service by creating a route" procedure.
 
 .Prerequisites
 
-* Install the `oc` CLI and log in as a cluster administrator.
+* You installed the {oc-first} and logged in as a cluster administrator.
 
 .Procedure
 
-. Create a new project for your service by running the `oc new-project` command:
+. Create a new project for your service by entering the following `oc new-project` command in your CLI:
 +
 [source,terminal]
 ----
-$ oc new-project myproject
+$ oc new-project <service_project_name> <1>
 ----
 
-. Use the `oc new-app` command to create your service:
+. Enter the `oc new-app` command to create your service:
 +
 [source,terminal]
 ----
-$ oc new-app nodejs:12~https://github.com/sclorg/nodejs-ex.git
+$ oc new-app <service_name>
 ----
 
-. To verify that the service was created, run the following command:
+. To patch the created `NodePort` service as a service that selects an available port on each node in your cluster, enter the following command. The command does not require that you define a route, router, or ingress controller for a cluster.
 +
 [source,terminal]
 ----
-$ oc get svc -n myproject
+$ oc patch svc/<service_name> --type='merge' -p='{"spec":{"type":"NodePort"}}' -n <service_project_name>
```

Comment:
the patching of this service with a node port does not belong here.  that is already covered in this section: https://79812--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-nodeport

Reply:
so basically, everything in the 4.16 docs in chapter 28.3.x is okay as it stands: https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html/networking/configuring-ingress-cluster-traffic#configuring-ingress-cluster-traffic-ingress-controller